docs/endpoints/configs/ospool.yaml

```diff
@@ -0,0 +1,34 @@
+display_name: OSPool
+engine:
+  type: GlobusComputeEngine
+  max_workers_per_node: 1
+
+  # This config uses apptainer containerization to ensure a consistent
+  # python environment on the worker side. Since apptainer limits writable
+  # directory paths, set working directory paths paths used by the worker to /tmp
+  worker_logdir_root: /tmp/logs
+  working_dir: /tmp/tasks_dir
```

Comment:
Confirming setup: this `/tmp` is specific to the container and is _not_ accessible to other users on the wider cluster or resource, correct?

Reply:
I see the security concern here. I confirmed with OSG support staff that these files will not show up on the host filesystem so this is not an issue.

Reply:
*nod; good deal.  That's not immediately obvious from this configuration; suggest making some mention of the security cover in the comment.

Reply:
Nice.